src/pages/en/guides/integrations-guide/image.md

```diff
@@ -496,7 +492,6 @@ const imageUrl = 'https://www.google.com/images/branding/googlelogo/2x/googlelog
 
 ## Troubleshooting
 
-*   If your installation doesn't seem to be working, make sure to restart the dev server.
```

Comment:
Are you SURE you want to take this one out? This is probably still good "troubleshooting" advice?

"If your installation doesn't seem to be working, you can try restarting the dev server."  ??

Reply:
Was on the edge removing this, so I'm happy to add it back!